main.py

```diff
@@ -10,7 +10,7 @@
 def main():
     load_dotenv() # only on local run
     print(os.environ)
-    queries_list = os.environ['INPUT_QUERIES'].split(';')
+    queries_list = map(str.strip, os.environ['INPUT_QUERIES'].split(';'))
```

Comment:
why is this required?

Reply:
Just for safety to get rid of unnecessary whitespace